<fix>[compute]: support CreateVmInstanceAction.diskAO.systemTags

zstack-robot-1 opened the pull request:
Resolves: ZSV-9791
Related: ZSV-9750
Related: ZSV-8585

Change-Id: I73716165686d6b7376627178616d65796d676c76

sync from gitlab !8321

Commit:
<fix>[compute]: support CreateVmInstanceAction.diskAO.systemTags

Resolves: ZSV-9791
Related: ZSV-9750
Related: ZSV-8585

Change-Id: I73716165686d6b7376627178616d65796d676c76

MatheMatrix commented:
**Comment from wenhao.zhang:**

报出的未通过 case 是 "virtio::dirver" tag 相关。

这类 tag 应该被标记在 VmInstanceVO 上，放在 diskAO 上会导致错误

Commit:
<fix>[compute]: clean "driver::virtio" tag when validate CreateVmAction

"driver::virtio" is tag for VmInstanceVO (not for VolumeVO)

Related: ZSV-9791
Related: ZSV-8585

Change-Id: I66726575787a68747a666c666c7a746d6c6e6d63

Commit:
<fix>[compute]: split diskAO to rootDisk and dataDisks

Resolves: ZSV-8585

Change-Id: I766b6f7462716f6a6a666161697377616a636875
